### DISCUSSION TITLE
Do  2dii.* packages have anything  we can reuse?

### DISCUSSION THREAD

**Discussion by maurolepore** [2DegreesInvesting/tiltIndicator#130] [Q&A] [answered]
The package tilt.company.match overlapped with [r2dii.match](https://rmi-pacta.github.io/r2dii.match) but didn't use it -- except for one feature. That was surprising to @AnneSchoenauer and @Tilmon so I better remind you here again.

Do 2dii.* packages have anything useful for tiltIndicator?

For example:

1. Is there anything useful in [r2dii.data]( https://rmi-pacta.github.io/r2dii.data/reference/index.html)? Maybe WEO data?

``` r
r2dii.data::region_isos |> tail()
#>             region isos   source
#> 5920        brazil   br weo_2020
#> 5921         india   in weo_2020
#> 5922         japan   jp weo_2020
#> 5923        russia   ru weo_2020
#> 5924  south africa   za weo_2020
#> 5925 united states   us weo_2020

r2dii.data::region_isos |> head()
#>               region isos   source
#> 1 advanced economies   bg weo_2019
#> 2 advanced economies   hr weo_2019
#> 3 advanced economies   cy weo_2019
#> 4 advanced economies   mt weo_2019
#> 5 advanced economies   ro weo_2019
#> 6 advanced economies   au weo_2019
```

4. Is there any useful calculation or idea in [r2dii.analysis](https://rmi-pacta.github.io/r2dii.analysis/index.html)?

5. Is there any useful plot or idea for a plot in [r2dii.plot](https://rmi-pacta.github.io/r2dii.plot/index.html)?

**Comment by maurolepore** [comment]
cc @Tilmon @lindadelacombaz

**Comment by Tilmon** [comment, marked as answer]
@maurolepore does this thread relate to the v1 indicators, i.e. do you think we should still review it for the delivery end of this current v1? Or  are you rather flagging it for the long-term? Because at least for the short term we already have everything that we need, don't we? 
Probably good idea though to look at it for the long-term...

**Reply by AnneSchoenauer** [reply]
@Tilmon can we put it somewhere so that we don't forget for version 2 though? I was skimming through it and I think it makes a lot of sense! :)

**Reply by Tilmon** [reply]
@AnneSchoenauer @maurolepore I put it down as ticket for end of May: [Do 2dii.* packages have anything we can reuse?#203](https://github.com/2DegreesInvesting/tiltIndicator/issues/203)

**Comment by maurolepore** [comment]
Thanks. OK, happy to go with that we have  now.
